h2o-py/h2o/group_by.py

```diff
@@ -50,52 +73,143 @@ def __init__(self, fr, by):
 
 
     def min(self, col=None, na="all"):
+        """
+        Calculate the minimum (per numerical column) of each group of a GroupBy object for column(s) specified by
+        the user.
```

Comment:
This is rather confusing. You probably wanted to say "Calculate the minimum of column(s) `col` within each group of the GroupBy object; if no `col` is given then compute the minimum among all numeric columns other than those being grouped on".